main.py

```diff
@@ -1,22 +1,37 @@
-from flask import Flask, request
-from flask_json import FlaskJSON
+import json
+from flask import Flask, request, make_response
 
-app = Flask(__name__)
-FlaskJSON(app)
 
-@app.route("/")
-def hello():
-    return "Hello World!"
-
-@app.route('/login', methods=['POST'])
-def login():
+def create_app():
+    app = Flask(__name__)
     
-    data = request.get_json(force=True)
+    @app.route('/login', methods=['POST'])
```

Comment:
You probably don't want to define the method inside `create_app`, do it like you did it before.